Classes/Mautic/AuthorizationFactory.php

```diff
@@ -21,8 +21,6 @@
 
 class AuthorizationFactory implements SingletonInterface
 {
-    public const VERSION = 'OAuth1a';
```

Comment:
This is a breaking change since other extensions may use this constant. Please consider to mark this constant as deprecated and remove it with the next major release